client/landing/gutenboarding/hooks/use-selected-plan.ts

```diff
@@ -50,7 +51,21 @@ export function useHasPaidPlanFromPath() {
 	return planFromPath && ! isPlanFree( planFromPath?.storeSlug );
 }
 
-export function useShouldSiteBePublic() {
+export function useNewSitePublicOption() {
 	const currentSlug = useSelectedPlan()?.storeSlug;
-	return useSelect( ( select ) => select( PLANS_STORE ).isPlanEcommerce( currentSlug ) );
+	const isEcommerce = useSelect( ( select ) =>
+		select( PLANS_STORE ).isPlanEcommerce( currentSlug )
+	);
+
+	if ( isEcommerce ) {
+		return 1; // Public and discoverable
```

Comment:
I think you should use a named constant here instead of the number and comment e.g.
```
return PRIVACY_MODES.PUBLIC_AND_DISCOVERABLE
```
I see that the API defines the meaning of those numbers but it would be better to have something more formal here I think

Reply:
Good call. I could define those constants in the `@automattic/data-stores` package and [use them in the API type definition too](https://github.com/Automattic/wp-calypso/blob/master/packages/data-stores/src/site/types.ts#L37).